api/src/header/header.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { Controller, Get, Param } from '@nestjs/common';
+import { HeaderService } from './header.service';
+import { Header } from './entity/header.entity';
+
+@Controller('header')
+export class HeaderController {
+  constructor(private readonly headerService: HeaderService) {}
+
+  @Get()
+  async findAll(): Promise<Header[]> {
+    return this.headerService.findAll();
+  }
+
+  @Get(':id')
```

Comment:
same concept, this is a single type not a collection of resources, you don't need this route.

Reply:
the get by id ?
so if we created a single document for everyone in the team how would we retrieve the needed one?
or we will git one document for each section and that's it

Reply:
the whole design and development based on one person and one database, we never discussed this site for multiple users.

Reply:
for that i requested a meeting 
please arrange one ASAP